packages/react-router/index.ts

```diff
@@ -277,7 +277,7 @@ export type {
   MiddlewareEnabled as UNSAFE_MiddlewareEnabled,
 } from "./lib/types/future.ts";
 export type { unstable_SerializesTo } from "./lib/types/serializes-to.ts";
-export type { Register } from "./lib/types/register";
+export type { Register, Pages, RouteFiles } from "./lib/types/register";
```

Comment:
Now exported as public API from `react-router` for consumption by `react-router/internal`